uma-rs-resteasy/src/main/java/org/gluu/oxd/rs/protect/resteasy/ResourceRegistrar.java

```diff
@@ -91,6 +91,15 @@ private void register(RsResource rsResource) {
                 } else {
                     resource.setScopes(condition.getScopes());
                 }
+                //set creation and expiration timestamp
+                if (safeToInt(rsResource.getIat()) && safeToInt(rsResource.getExp())) {
```

Comment:
it should be valid to set just `exp`. `iat` will be set by AS.

Reply:
done